content/1.7/back-office/shop-parameters/contact/stores/add-edit stores.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Add / Edit Stores
+weight: 1
+---
+
+# Add / Edit Store
+
+When entered the add or editing current store the following fields are displayed:
+
+1) Name
+
+Input field for the name of the store. Can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+
+3) Address
+
+Input field for the address of the shop. This field is mandatory and can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+If there is no address input, when trying to save shows the message: "The field address1 is required at least in English (English)."
+
+5) Address (2)
+
+Second input field for the address of the shop. This field can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+
+7) Zip/Postal code
+
+Input field for the zip/postal code of the shop.
+
+9) City
+
+Input field for the city of the shop, this field is mandatory.
+When trying to save without a city entered, shows the message: "The city field is required."
+
+11) Country
+
+Drop-down menu for the country of the shop, this is a mandatory selection field.
+
+13) State (is shown only when the country is selected as "United states")
+
+Drop-down menu for the state of the shop, this is a mandatory selection field.
+
+15) Latitude/Longtitude
+
+These are 2 separate input fields for the lattitude and longtitude of the shop, this field is mandatory.
+Only numbers are allowed for these fields. When trying to input anything but not numbers, a message is shown: "Latitude and longitude are required." Also, the numbers in the fields are limited, if there are more numbers than allowed, messaged are shown: 
+"There are 2 errors.
+The latitude field is too long (13 chars max).
+The longitude field is too long (13 chars max)."
+
+17) Phone
+
+Input field for the phone of the shop.
+
+19) Fax
+
+Input field for the fax of the shop.
+
+21) Email address
+
+Input field for the email of the shop.
+
+23) Note
+
+Input field for the notes of the shop.
+
+25) Active
+
+Toggle switch for "yes" or "no"
+
```

Comment:
What is the default state while adding a store ? Is it on Yes or No ?
What happens in front-office if it's not active ?

When the label is hovered, a tooltip is displayed. I let you specify this.